doc/radosgw/lua-scripting.rst

```diff
@@ -298,6 +298,9 @@ Request Fields
 +----------------------------------------------------+----------+--------------------------------------------------------------+----------+-----------+----------+
 | ``Request.HTTP.Domain``                            | string   | domain name                                                  | no       | no        | no       |
 +----------------------------------------------------+----------+--------------------------------------------------------------+----------+-----------+----------+
+| ``Request.HTTP.Environment``                       | table    | string to string metadata map. holding a complete list of    | yes      | yes       | no       |
```

Comment:
string-to-string metadata map that holds a complete list of all HTTP headers sent in the request

Reply:
fixed